src/views/Profile/components/ClaimPointsCallout .tsx

```diff
@@ -54,7 +54,9 @@ const ClaimPointsCallout = () => {
         <Flex flexDirection={['column', null, 'row']} justifyContent={['start', null, 'space-between']}>
           <Flex alignItems="center" mb={['16px', null, 0]}>
             <PrizeIcon width="32px" mr="8px" />
-            <Heading size="lg">{t(`${totalPointsToCollect} Points to Collect`, { num: totalPointsToCollect })}</Heading>
+            <Heading scale="lg">
+              {t(`${totalPointsToCollect} Points to Collect`, { num: totalPointsToCollect })}
```

Comment:
This won't work. Needs to be
```
{t(`%num% Points to Collect`, { num: totalPointsToCollect })}
```

Reply:
Please add the key to `translations.json` as well.

Reply:
Nice spot.